src/main/java/uk/gov/companieshouse/api/accounts/model/filing/Link.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.companieshouse.api.accounts.model.filing;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Link {
+
+    @JsonProperty("rel")
+    private String relationship;
+
+    @JsonProperty("href")
+    private String link;
```

Comment:
Should we rename the `@JsonProperty("href")` to be `link`?  or `link` to be `href`? Just wondering if there is a specific reason they don't match?

Reply:
What I can do is to change the variable's name to be `href` instead...as i dont want to change the model.